packages/tokens/styles/color.scss

```diff
@@ -0,0 +1,32 @@
+$white: #fff;
+$black: #000;
+
+// Primary colours
+$ca-palette-ink: #3e4543;
+$ca-palette-paper: #f2edde;
+$ca-palette-coral: #f04c5d;
+
+// Secondary colours
```

Comment:
@ActuallyACat nit: we're moving away from these groupings of primary, secondary, tertiary for our colors so you could leave out the comments here